spec/menu_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'menu'
+
+describe Menu do
+
+  let(:dish_klass) {double(:dish_klass)}
+  let(:dish) {double(:dish)}
+  let(:dish2) {double(:dish)}
+  let(:dish_list) { [ {name: :name_arg, price: :price_arg},
+                      {name: :name_arg, price: :price_arg} ]
+                    }
```

Comment:
Lint/BlockAlignment: `end` at 10, 20 is not aligned with `let(:dish_list) { [ {name: :name_arg, price: :price_arg},` at 8, 2.